### PR TITLE
Add initial README.md

### PR DESCRIPTION
**Pull request by svrnm**
This is an initial proposal for a README.md, I expect this to change significantly over time, but I also think that it would be good to have something to start from and something people can find when they review our repository

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,17 @@
...
+The goal of this SIG is to improve the experience of contributors to the OpenTelemetry project. 
+The SIG creates programs and processes and consults other SIGs to improve the overall community health. 
+
+## Get Involved
+
+There are many ways to get involved in our work:
+
+- Create [issues]([github.com/open-telemetry/sig-contributor-experience/issues/](https://github.com/open-telemetry/sig-contributor-experience/issues/new)), comment on [existing issues](github.com/open-telemetry/sig-contributor-experience/issues/), [review pull requests](https://github.com/open-telemetry/sig-contributor-experience/pulls)
+- Get in touch with us on the [CNCF slack](https://slack.cncf.io/) channel [#otel-contributor-experience](https://cloud-native.slack.com/archives/C06TMJ2R0SK)
+- [Attend on of our meetings](https://docs.google.com/document/d/1CTQI0p3QF8JP8reV8z_ggcs8KE5YVPpQGvAQknw4qP0/edit#heading=h.x3xl5syz15zx), we alternate between two times to make it possible for you to join no matter which time zone you live or work in!
```
Comment (mx-psi) [inline README.md:14]:
```suggestion
- [Attend one of our meetings](https://docs.google.com/document/d/1CTQI0p3QF8JP8reV8z_ggcs8KE5YVPpQGvAQknw4qP0/edit#heading=h.x3xl5syz15zx), we alternate between two times to make it possible for you to join no matter which time zone you live or work in! 
```

**Review by mx-psi — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,2 +1,17 @@
...
+## Get Involved
+
+There are many ways to get involved in our work:
+
+- Create [issues]([github.com/open-telemetry/sig-contributor-experience/issues/](https://github.com/open-telemetry/sig-contributor-experience/issues/new)), comment on [existing issues](github.com/open-telemetry/sig-contributor-experience/issues/), [review pull requests](https://github.com/open-telemetry/sig-contributor-experience/pulls)
+- Get in touch with us on the [CNCF slack](https://slack.cncf.io/) channel [#otel-contributor-experience](https://cloud-native.slack.com/archives/C06TMJ2R0SK)
+- [Attend one of our meetings](https://docs.google.com/document/d/1CTQI0p3QF8JP8reV8z_ggcs8KE5YVPpQGvAQknw4qP0/edit#heading=h.x3xl5syz15zx), we alternate between two times to make it possible for you to join no matter which time zone you live or work in! 
+
+> [!NOTE]
+> This SIG operates **async** by default, attending any meetings is not mandatory! Feel free to choose your preferred way of communication
```
Comment (kaylareopelle) [inline README.md:17]:
```suggestion
> This SIG operates **async** by default, attending meetings is not mandatory! Feel free to choose your preferred way of communication.
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,17 @@
...
+## What is the Contributor Experience SIG?
+
+The goal of this SIG is to improve the experience of contributors to the OpenTelemetry project. 
+The SIG creates programs and processes and consults other SIGs to improve the overall community health. 
+
+## Get Involved
+
+There are many ways to get involved in our work:
+
+- Create [issues]([github.com/open-telemetry/sig-contributor-experience/issues/](https://github.com/open-telemetry/sig-contributor-experience/issues/new)), comment on [existing issues](github.com/open-telemetry/sig-contributor-experience/issues/), [review pull requests](https://github.com/open-telemetry/sig-contributor-experience/pulls)
```
Comment (tiffany76) [inline README.md:12]:
```suggestion
- Create [issues]([github.com/open-telemetry/sig-contributor-experience/issues/](https://github.com/open-telemetry/sig-contributor-experience/issues/new)), comment on [existing issues](github.com/open-telemetry/sig-contributor-experience/issues/), or [review pull requests](https://github.com/open-telemetry/sig-contributor-experience/pulls).
```

File: README.md [hunk 4]
```diff
@@ -1,2 +1,17 @@
...
+- Get in touch with us on the [CNCF slack](https://slack.cncf.io/) channel [#otel-contributor-experience](https://cloud-native.slack.com/archives/C06TMJ2R0SK)
```
Comment (tiffany76) [inline README.md:13]:
```suggestion
- Get in touch with us on the [CNCF slack](https://slack.cncf.io/) channel [#otel-contributor-experience](https://cloud-native.slack.com/archives/C06TMJ2R0SK).
```

File: README.md [hunk 5]
```diff
@@ -1,2 +1,17 @@
...
+- [Attend one of our meetings](https://docs.google.com/document/d/1CTQI0p3QF8JP8reV8z_ggcs8KE5YVPpQGvAQknw4qP0/edit#heading=h.x3xl5syz15zx), we alternate between two times to make it possible for you to join no matter which time zone you live or work in!
```
Comment (tiffany76) [inline README.md:14]:
```suggestion
- [Attend one of our meetings](https://docs.google.com/document/d/1CTQI0p3QF8JP8reV8z_ggcs8KE5YVPpQGvAQknw4qP0/edit#heading=h.x3xl5syz15zx). We alternate between two times to make it possible for you to join no matter which time zone you live or work in! 
```

File: README.md (hunk 2 above)
Comment (tiffany76) [inline README.md:17]:
```suggestion
> This SIG operates **async** by default, so attending any meetings is not mandatory! Feel free to choose your preferred way of communication
```

**Review by tiffany76 — COMMENTED** [review]
Just a few punctuation and conjunction suggestions. :+1:

File: README.md [hunk 6]
```diff
@@ -1,2 +1,17 @@
-# sig-contributor-experience
-TODO
+# <img src="https://opentelemetry.io/img/logos/opentelemetry-logo-nav.png" alt="OpenTelemetry Icon" width="45" height=""> OpenTelemetry Contributor Experience SIG
+
+## What is the Contributor Experience SIG?
+
+The goal of this SIG is to improve the experience of contributors to the OpenTelemetry project. 
+The SIG creates programs and processes and consults other SIGs to improve the overall community health.
```
Comment (codefromthecrypt) [inline README.md:6]:
```suggestion
To achieve this, we create programs and processes around key contribution stages, such as first commit, or the transition from regular maintainer to approver.  We also consult with other SIGs to feedback into the process with an aim to improve overall community health.
```

**Review by codefromthecrypt — APPROVED** [review]
Understood if too wordy, but the intents here are..

* "we" intentional use of we, so that the reader thinks they are a part of this from sentence 2.
* light examples to make more concrete what exactly is happening here, so it doesn't feel first about process (it is, but it is also about outcomes)

hope this helps!

File: README.md (hunk 2 above)
Comment (svrnm) [reply README.md:17]:
```suggestion
> This SIG operates **async** by default, so attending meetings is not mandatory! Feel free to choose your preferred way of communication.
```

Comment (svrnm) [reply README.md:17]: merging @tiffany76's and @kaylareopelle's suggestions

**Review by codefromthecrypt — APPROVED** [review]

File: README.md [hunk 7]
```diff
@@ -1,2 +1,17 @@
-# sig-contributor-experience
-TODO
+# <img src="https://opentelemetry.io/img/logos/opentelemetry-logo-nav.png" alt="OpenTelemetry Icon" width="45" height=""> OpenTelemetry Contributor Experience SIG
+
+## What is the Contributor Experience SIG?
+
+The goal of this SIG is to improve the experience of contributors to the OpenTelemetry project. 
+To achieve this, we create programs and processes around key contribution stages, such as first commit, or the transition from regular maintainer to approver.  We also consult with other SIGs to feedback into the process with an aim to improve overall community health.
+
+## Get Involved
```
Comment (theletterf) [inline README.md:8]:
```suggestion
## Get involved
```

File: README.md [hunk 8]
```diff
@@ -1,2 +1,17 @@
...
+
+There are many ways to get involved in our work:
```
Comment (theletterf) [inline README.md:10]:
```suggestion
There are many ways to get involved:
```

File: README.md [hunk 9]
```diff
@@ -1,2 +1,17 @@
...
+
+- Create [issues]([github.com/open-telemetry/sig-contributor-experience/issues/](https://github.com/open-telemetry/sig-contributor-experience/issues/new)), comment on [existing issues](github.com/open-telemetry/sig-contributor-experience/issues/), or [review pull requests](https://github.com/open-telemetry/sig-contributor-experience/pulls).
```
Comment (theletterf) [inline README.md:12]: Consider splitting into three bullet list items.

File: README.md [hunk 10]
```diff
@@ -1,2 +1,17 @@
-# sig-contributor-experience
-TODO
+# <img src="https://opentelemetry.io/img/logos/opentelemetry-logo-nav.png" alt="OpenTelemetry Icon" width="45" height=""> OpenTelemetry Contributor Experience SIG
+
+## What is the Contributor Experience SIG?
+
+The goal of this SIG is to improve the experience of contributors to the OpenTelemetry project.
```
Comment (theletterf) [inline README.md:5]: Project or projects, as in all SIG code projects on GH?

**Review by theletterf — APPROVED** [review]

File: README.md (hunk 10 above)
Comment (svrnm) [reply README.md:5]:
```suggestion
The goal of this SIG is to improve the experience of contributors to the OpenTelemetry Project. 
```

Comment (svrnm) [reply README.md:5]: otel as a project itself, does making the p uppercase clarify that?

File: README.md (hunk 9 above)
Comment (svrnm) [reply README.md:12]: I wanted to avoid having a sub list of items, it looks convoluted in the markdown, because of the links, but the rich view is ok

File: README.md [hunk 11]
```diff
@@ -1,2 +1,17 @@
...
+## What is the Contributor Experience SIG?
+
+The goal of this SIG is to improve the experience of contributors to the OpenTelemetry project. 
+To achieve this, we create programs and processes around key contribution stages, such as first commit, or the transition from regular maintainer to approver.  We also consult with other SIGs to feedback into the process with an aim to improve overall community health.
+
+## Get involved
+
+There are many ways to get involved:
+
+- Create [issues]([github.com/open-telemetry/sig-contributor-experience/issues/](https://github.com/open-telemetry/sig-contributor-experience/issues/new)), comment on [existing issues](github.com/open-telemetry/sig-contributor-experience/issues/), or [review pull requests](https://github.com/open-telemetry/sig-contributor-experience/pulls).
```
Comment (kaylareopelle) [inline README.md:12]:
```suggestion
- Create [issues](https://github.com/open-telemetry/sig-contributor-experience/issues/new), comment on [existing issues](https://github.com/open-telemetry/sig-contributor-experience/issues/), or [review pull requests](https://github.com/open-telemetry/sig-contributor-experience/pulls).
```

**Review by tiffany76 — APPROVED** [review]

File: README.md [hunk 12]
```diff
@@ -1,2 +1,17 @@
-# sig-contributor-experience
-TODO
+# <img src="https://opentelemetry.io/img/logos/opentelemetry-logo-nav.png" alt="OpenTelemetry Icon" width="45" height=""> OpenTelemetry Contributor Experience SIG
+
+## What is the Contributor Experience SIG?
+
+The goal of this SIG is to improve the experience of contributors to the OpenTelemetry project. 
+To achieve this, we create programs and processes around key contribution stages, such as first commit, or the transition from regular maintainer to approver.  We also consult with other SIGs to feedback into the process with an aim to improve overall community health.
```
Comment (ChrsMark) [inline README.md:6]: This sounds a bit odd (is the intention to highligiht a "downgrade" of role from maintainer to approver?). 
Maybe it should be:
```suggestion
To achieve this, we create programs and processes around key contribution stages, such as first commit, or the transition from regular contributor to approver.  We also consult with other SIGs to feedback into the process with an aim to improve overall community health.
```

Comment (svrnm) [reply README.md:6]: ah, lol... brain not good! Thanks

**Comment by svrnm** [comment]
thank you all for your reviews! I merged this now, it's an iterative process to get this right, so please go ahead and raise additional PRs to get this README.md from "OK" to "good" to "great" :-)
